_site/blog/welcome-to-jekyll/index.html

```diff
@@ -206,7 +209,7 @@ <h6>Find more Red Hat Middleware Community Projects</h6>
           
             <li><a href="http://teiid.jboss.org/" target="_blank">Teiid</a></li>
           
-            <li><a href="http://wildfly-swarm.io/" target="_blank">WildFly Swarm</a></li>
+            <li><a href="https://thorntail.io/posts/the-end-of-an-era/" target="_blank">WildFly Swarm</a></li>
```

Comment:
<li><a href="https://thorntail.io/posts/the-end-of-an-era/" target="_blank">Thorntail</a></li>
```suggestion
            <li><a href="https://thorntail.io/posts/the-end-of-an-era/" target="_blank">Thorntail</a></li>
```